tasks/install-Debian.yml

```diff
@@ -19,3 +19,8 @@
   package:
     name: rake
     state: present
+
+- name: Install phantomjs
```

Comment:
this. as well as `rake`, should have a comment why phantomjs is installed because not all jenkins users would not expect it to be installed.